test/e2e/volume_provisioning.go

```diff
@@ -513,7 +512,7 @@ func newStorageClass(pluginName, suffix string) *storage.StorageClass {
 			Kind: "StorageClass",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name: "myclass-" + suffix,
+			GenerateName: "sc-" + suffix + "-",
```

Comment:
`suffix` is sometimes passed in empty. Is it ok for the generated name to begin with "sc--"?